fs_src/index.html

```diff
@@ -236,12 +236,22 @@ <h1 id="head">Switch</h1>
         <input type="text" id="name">
       </div>
       <div class="form-control">
-        <label>HAP Type:</label>
+        <label>HAP Service Type:</label>
         <select id="svc_type">
           <option id="svc_type_-1" value="-1">Disabled</option>
           <option id="svc_type_0" value="0">Switch</option>
           <option id="svc_type_1" value="1">Outlet</option>
           <option id="svc_type_2" value="2">Lock</option>
+          <option id="svc_type_3" value="3">Valve</option>
+        </select>
+      </div>
+      <div class="form-control" id="char_type_container" style="display: none">

```

Comment:
it's valve type, let's call it that.

Reply:
I chose the generic name to make them reusable for other cases as well.
- Security System Alarm Type
- Slat Type

Reply:
they will have different nomenclature though. we'll use different parameter for these, if and when they are implemented.